<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowLogin</class>
 <widget class="QMainWindow" name="WindowLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>470</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/imagenes/arbolde-sakura-en-japon-anime-zuhr2aledzyd5wso.webp);
border-top-left-radius: 50px;</string>
     </property>
     <property name="text">
      <string>l</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>290</width>
       <height>470</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">

background-color: rgba(255, 255, 255, 1); /* Fondo blanco */  
    border-bottom-right-radius: 70px; /* Esquina inferior derecha redondeada */  
    box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.2); /* Sombra */  
    padding: 20px; </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Niagara Engraved</family>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PASSKEEPER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GeoSlab703 MdCn BT</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>INICIE SESION</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_usuario">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>200</y>
       <width>195</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
    background-color: rgba(0, 0, 0, 0); /* Fondo transparente */  
    border: none; /* Sin borde por defecto */  
    border-bottom: 2px solid rgba(46, 82, 101, 0.8); /* Borde inferior con color */  
    color: rgba(0, 0, 0, 0.94); /* Color del texto */  
    padding-bottom: 20px; /* Espacio inferior dentro del QLineEdit */  
</string>
     </property>
     <property name="placeholderText">
      <string>  USUARIO</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_contrasena">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>195</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
    background-color: rgba(0, 0, 0, 0); /* Fondo transparente */  
    border: none; /* Sin borde por defecto */  
    border-bottom: 2px solid rgba(46, 82, 101, 0.8); /* Borde inferior con color */  
    color: rgba(0, 0, 0, 0.94); /* Color del texto */  
    padding-bottom: 20px; /* Espacio inferior dentro del QLineEdit */  
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>  CONTRASEÑA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_iniciarSesion">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>320</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
    background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,   
                                       stop: 0 rgba(135, 206, 250, 255),   
                                       stop: 1 rgba(70, 130, 180, 255)); /* Degradado celeste */  
    color: rgba(255, 255, 255, 240); /* Color del texto */  
    border: none; /* Sin borde */  
    padding: 10px; /* Espaciado interno del botón */  
    border-radius: 5px; /* Esquinas redondeadas (opcional) */  

</string>
     </property>
     <property name="text">
      <string>INICIAR SESION</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>380</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>Nuevo usuario??</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_registrarse">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>400</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
    background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,   
                                       stop: 0 rgba(144, 238, 144, 255), /* Verde claro */  
                                       stop: 1 rgba(34, 139, 34, 255)); /* Verde oscuro */  
    color: rgba(255, 255, 255, 240); /* Color del texto */  
    border: none; /* Sin borde */  
    padding: 10px; /* Espaciado interno del botón */  
    border-radius: 5px; /* Esquinas redondeadas */  

</string>
     </property>
     <property name="text">
      <string>REGISTRARSE</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../otros/imagenes/res.qrc"/>
 </resources>
 <connections/>
</ui>
